packages/cli/src/commands/init/init.js

```diff
@@ -160,16 +216,21 @@ export default (async function initialize(
    */
   const version: string = minimist(process.argv).version || 'latest';
 
-  if (fs.existsSync(projectName)) {
-    throw new DirectoryAlreadyExistsError(projectName);
-  }
+  const projectDirectoryExists = await isProjectDirectoryExistent({
```

Comment:
Not very fan of calling this twice, not sure if it would be a better option to pass the value of this to `createProject` and then to `createFromTemplate`, would appreciate other opinions.